### ISSUE TITLE
run_script plan function fails with 'wrong number of arguments (2 for 3)'

### ISSUE THREAD

**Issue by ktreese** [puppetlabs/bolt#155]
```
➜  bolt --version
0.7.0
```

```
➜  cat ./modules/tasks_demo/plans/puppet.pp 
plan tasks_demo::puppet(
  String $nodes,
  String $service_action = 'status',
  String $service_name = 'puppet',
) {

  $nodes_array = split($nodes, ',')

  run_script('tasks_demo/install_puppet_5_agent.sh', $nodes_array)
  #run_command('/opt/puppetlabs/bin/puppet --version', $nodes_array)
  #run_task('service', $nodes_array, action => $service_action, name => $service_name)

}
```

```
➜  bolt plan run tasks_demo::puppet nodes=127.0.0.1:32768,127.0.0.1:32769,127.0.0.1:32770 --modulepath ./modules -u root -p root
Error: Evaluation Error: Error while evaluating a Function Call, wrong number of arguments (2 for 3) at /Users/ktreese/workspace/puppet/tasks/modules/tasks_demo/plans/puppet.pp:9:3 on node macbook
Exiting because of an error in Puppet code
➜   
```

```
➜  ls -l ./modules/tasks_demo/                    
total 0
drwxr-xr-x  4 reesek  1575811233  136 Nov  9 16:08 files
drwxr-xr-x  6 reesek  1575811233  204 Nov  9 16:08 plans
drwxr-xr-x  6 reesek  1575811233  204 Nov  8 11:14 tasks
➜  ls -l ./modules/tasks_demo/files 
total 40
-rw-r--r--  1 reesek  1575811233   2889 Nov  9 16:08 bashcheck
-rw-r--r--  1 reesek  1575811233  14233 Nov  9 15:40 install_puppet_5_agent.sh
```

[Documentation](https://puppet.com/docs/bolt/0.x/writing_plans.html#calling-basic-plan-functions) indicates two positional arguments.

**Comment by ktreese** [comment]
Note: Downgrading to bolt 0.6.1 does not present the same error.

**Comment by Iristyle** [comment]
@ktreese yes, this is an oversight that has been fixed in the vendored copy of Puppet - see https://github.com/puppetlabs/puppet/pull/6347 / https://tickets.puppetlabs.com/browse/PUP-8137

I believe this will be shipped shortly in the upcoming 0.8.0 release, so closing the issue.  Please re-open if for some reason your issue is not resolved at that point.

Thanks!
